code/datums/gamemodes/campaign/individual_stats.dm

```diff
@@ -412,10 +412,11 @@
 		return
 	stats.current_mob = owner //taking over ssd's creates a mismatch
 	//we have to update selected tab/job so we load the correct data for the UI
-	if(!isliving(owner))
+	var/mob/living/living_owner = owner
+
+	if(!isliving(owner) || !(living_owner.job.title in stats.valid_jobs))
```

Comment:
Is there a threat of this runtiming on null job?

Reply:
probably isn't possible but doesn't hurt to be sure